networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -134,5 +137,80 @@ TEST_F(DynBetweennessGTest, runApproxBetweenness) {
 	DEBUG("Number of samples: ", bc1.numberOfSamples());
 }
 
+TEST_F(DynBetweennessGTest, runDynVsStaticCaseInsertDirected){
+	srandom(0);
+	Aux::Random::setSeed(0, false);
+
+	auto genEdgeInsert = [](const Graph& g){
+		int n = g.numberOfNodes();
+		while(true){
+			int u = random() % n, v = random() % n;
+			if(u == v or g.hasEdge(u, v))continue;
+			return std::make_pair(u, v);
+		}
```

Comment:
This loop should be implemented as follows:
```
node u = g.randomNode();
node v = g.randomNode();
while (u == v || g.hasEdge(u, v))
    v = g.randomNode();
return std::make_pair(u, v);
```

Reply:
If the loop is implemented that way, and node u is full, that while would never end. Following your suggestion I will put it like:
```
node u = g.randomNode();
node v = g.randomNode();
while (u == v || g.hasEdge(u, v)){
    u = g.randomNode();
    v = g.randomNode();
}
return std::make_pair(u, v);
```
do you agree?

Reply:
You are right, I agree.